runbotics/runbotics-orchestrator-ui/src/src-landing/views/MarketplaceOfferView/MarketplaceOfferView.module.scss

```diff
@@ -0,0 +1,68 @@
+@import '#src-landing/scss/main';
+@import '#src-landing/components/Typography/Typography.module.scss';
+
+.blogWrapper {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    width: 100%;
+    max-width: $MAX_SECTION_CONTENT_WIDTH;
+
+    margin: 0 auto;
+    padding: $x-content-padding;
+    padding-top: calc($x-content-padding + $HEADER_HEIGHT);
+}
+
+@mixin link-styles {
+    color: $palette-accent-primary;
+    border-bottom: 2px solid transparent;
+    text-decoration: none;
+    transition: $landing-link-transition;
+
+    &:hover {
+        border-bottom-color: $palette-accent-primary;
+    }
+}
+
+.contentArticle {
+    max-width: $MAX_BLOG_CONTENT_WIDTH;
+
+    padding-top: $padding-5;
+    padding-bottom: 64px;
+
+    p > a {
+        @include link-styles;
+    }
+}
+
+.breadCrumbsWrapper {
+    width: 100%;
+    padding: 24px clamp(10px, 8vw, 140px);
+}
+
+.tagsWrapper {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;  
+    margin-top: $margin-5;
+}
+
+.tag {
+    margin-right: $margin-1;
+    border-bottom: 3px solid transparent;
+}
+
+.tagLink {
+    @extend .typography__primary, .typography__Roboto, .typography__h4;
```

Comment:
woa, never seen sth like that. What's that needed for?

Reply:
are these inherited classes written by us or generated by some MUI? if the first option alright, if second I'm worried about possible unexpected changes here or complexity